### PR TITLE
simplify PR template

### PR DESCRIPTION
**Pull request by jsteemann**
This PR simplifies our PR template, so that filling it out becomes less work.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,37 +1,22 @@
...
-*The below list is not exhaustive, think thoroughly whether the provided information is sufficient.*
-*Remove options that do not apply*
-
 ### Scope & Purpose
 
-*(Can you describe what functional change your PR is trying to effect?)*
+*(Please describe the changes of this PR for reviewers)*
 
-- [ ] Bug-Fix for *devel-branch* (i.e. no need for backports?)
-- [ ] Bug-Fix for a *released version* (did you remember to port this to all relevant release branches?)
```
Comment (goedderz) [inline PULL_REQUEST_TEMPLATE.md:22]: I did like these, for one to check whether backports are done, and to link all backport PRs here.

I don't need these two checkboxes as they were, but can we have something for backports?

Comment (jsteemann) [reply PULL_REQUEST_TEMPLATE.md:22]: Sure, let's add backports somewhere!

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,37 +1,22 @@
...
+*(Please reference tickets / specification etc)*
 
-- [ ] There is a GitHub Issue reported by a Community User: 
-- [ ] There is an internal planning ticket: 
-- [ ] There is a *JIRA Ticket number* (In case a customer was affected / involved): 
-- [ ] There is a design document: 
+- [ ] Docs PR: 
+- [ ] Enterprise PR:
+- [ ] GitHub issue / Jira ticket number:
+- [ ] Design document:
```
Comment (mchacki) [inline PULL_REQUEST_TEMPLATE.md:20]:
```suggestion

#### Backports:

- [ ] New functionality, no backports
- [ ] Backports required for:
```

Maybe like this?

Comment (jsteemann) [reply PULL_REQUEST_TEMPLATE.md:20]: Fine for me.

**Pull request merged**